mslib/mscolab/models.py

```diff
@@ -125,8 +125,10 @@ class Operation(db.Model):
     path = db.Column(db.String(255), unique=True)
     category = db.Column(db.String(255))
     description = db.Column(db.String(255))
+    active = db.Column(db.Boolean)
+    last_used = db.Column(db.DateTime)
 
-    def __init__(self, path, description, category="default"):
+    def __init__(self, path, description, last_used, category="default", active=True):
```

Comment:
here you need also the last_used=None

if last_used is None:

Reply:
I think there is no need for initialization of "last_used" in models, as during operation creation it will be initialized with dattime object.

Reply:
look at the failing tests,I shared on chat.

Reply:
I'll see that.